### ISSUE TITLE
Invalid syntax in json payload

### ISSUE THREAD

**Issue by jenschristianschroder** [MicrosoftDocs/powerapps-docs#2519]
Sample JSON payload is invalid.

A back slash has sneaked its way into the @odata.type element. See below.
```
 "Description": {
     "@odata.type": "Microsoft.Dynamics.CRM.Label",
     "LocalizedLabels": [
       {
         "\@odata.type": "Microsoft.Dynamics.CRM.LocalizedLabel",
         "Label": "Media Polymorphic Lookup",
         "LanguageCode": 1033
       }
     ],
```

This results in the following error:
`"An error occurred while validating input parameters: System.ArgumentException: Stream was not readable....`

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 52939b82-7a1e-7320-c0a4-be2ea7f59724
* Version Independent ID: c67af10a-57f8-b450-5e0f-6c3f7d22079a
* Content: [Use multi-table lookup columns (Microsoft Dataverse) - Power Apps](https://docs.microsoft.com/en-us/powerapps/developer/data-platform/webapi/multitable-lookup?branch=pr-en-us-4448)
* Content Source: [powerapps-docs/developer/data-platform/webapi/multitable-lookup.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/developer/data-platform/webapi/multitable-lookup.md)
* Service: **powerapps**
* GitHub Login: @NHelgren
* Microsoft Alias: **nhelgren**

**Comment by jenschristianschroder** [comment]
is being fixed in #2518

**Comment by phecke** [comment]
@jenschristianschroder  Thanks for catching that. Fixed.
